OpenTabletDriver.Web/Views/Wiki/Install/Linux.cshtml

```diff
@@ -124,3 +124,9 @@
         </codeblock>
     </li>
 </ol>
+<hr/>
+
+<h3 class="wiki-nav-item" id="post-install">
+    Post-Installation
+</h3>
+<p>After installation, you may be required to look at the console of the OpenTabletDriver to find any problems, then follow the <a href="/Wiki/FAQ/Linux">FAQ</a> before your tablet works.</p>
```

Comment:
Not sure I like this, needs reworded in a shorter manner.

Reply:
```suggestion
<p>You're advised to look at the console tab on OpenTabletDriver UX to catch any problems. Then if necessary, read through the <a href="/Wiki/FAQ/Linux">FAQ</a>.</p>
```
Is this better?

Reply:
Resolved.